modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
```

Comment:
So we set up the replicated variable to always store the relevant local information at [1]?